src/structuretoolkit/build/geometry.py

```
@@ -0,0 +1,89 @@
+    """
+    if not inplace:
+        structure = structure.copy()
+    if axis is None:
+        axis = slice(None)
+    for _ in range(iterations):
+        neigh = get_neighbors(structure, num_neighbors=1)
+        dd = neigh.distances[:, 0]
+        if dd.min() >= min_dist:
+            break
+
+        I = dd < min_dist
```

Ruff (0.15.9)

[error] Ambiguous variable name: `I`

(E741)